src/common/boards/nrf_PCA10175_1.0.0_54H20.json

```diff
@@ -0,0 +1,111 @@
+{
+    "board": {
+        "boardVersion": "PCA10175",
+        "boardRevision": "1.0.0",
+        "boardName": "nRF54H20 DK v1.0.0"
```

Comment:
1.0.0 is an assumption.

Not updating current 0.7.0 definition file https://github.com/NordicSemiconductor/pc-nrfconnect-board-configurator/blob/main/src/common/boards/nrf_PCA10175_0.7.0_54H20.json

Reply:
UPD. New config should be for all devices